Makefile

```diff
@@ -0,0 +1,78 @@
+COMPOSE := docker-compose -f test/cluster/docker-compose.yml
+
+.PHONY: all
+all: test
+
+.PHONY: ci
+ci: fmt-check check check-without-features clippy test build
```

Comment:
Apart from running static checks, this command also runs tests. Considering users that will have to run tests in a docker container, it would be nice if there were two versions of this target - one which runs tests in a docker container, and the other which doesn't.